phpstan.neon

```diff
@@ -1,2 +1,6 @@
 includes:
 	- vendor/gamee/php-code-checker-rules/phpstan.neon
+
+parameters:
+	ignoreErrors:
+		- '/Strict comparison using === between array and false will always evaluate to false\./'
```

Comment:
Could we reqrite the PHP code so that we don't have to make execptions in phpstan?

Reply:
Return value of `apache_request_headers` in PHP 8 was changed, it throws exception instead of returning false on error.

Reply:
🤔  I am OK with releasing a major version with a try-catch block (BC break) and no php exceptions. That would seem clean to me. WDYT?

Reply:
Yes I think that will work for me.

Reply:
Could you please provide a link to php doc where I can find a mention of the exception? Thanks a lot.

Reply:
It looks like this change is not documented well, but you can check source: https://github.com/php/php-src/blob/master/sapi/cgi/cgi_main.c#L1580
I'm not familiar with C but it looks like PHPStan is right https://phpstan.org/r/53efc3c6-c53b-4752-bb34-ae6dd5141516

Reply:
I am a bit sceptical about merging a coded that is not documented on php.net. :/
Thanks for the MR but I will leave it open for now.